redfish-core/lib/processor.hpp

```diff
@@ -1462,6 +1613,10 @@ inline void requestRoutesProcessor(App& app)
                                                asyncResp, processorId,
                                                appliedConfigUri));
         }
+        if (locationIndicatorActive)
+        {
+            setProcessorObject(asyncResp, processorId, locationIndicatorActive);
+        }
```

Comment:
Perhaps wouldn't the body of patch (L1572-L1620) be be better if it is factored into a separate function?

Reply:
Absolutely, that makes it much easier to read. Thank you for the idea. I've changed it.